addons/danger/functions/fnc_brainForced.sqf

```diff
@@ -41,9 +41,11 @@ if (fleeing _unit) exitWith {
 
 // attack speed and stance
 if ((currentCommand _unit) isEqualTo "ATTACK") then {
-    _unit setVariable [QEGVAR(main,currentTask), "Attacking", EGVAR(main,debug_functions)];
-    [_unit, getAttackTarget _unit] call EFUNC(main,doAssaultSpeed);
+    private _attackTarget = getAttackTarget _unit;
+    if ((typeOf _attackTarget) isEqualTo "SuppressTarget") exitWith {deleteVehicle _attackTarget;};
```

Comment:
unsure if that will cause issue along the way. At some point an ai will end up with `objNull` as target. I hope we have checks for that in place, else they will fire or look at [0,0,0]

Reply:
Testing on nopryl server.

Reply:
Mind you. In my experience, if an 'ATTACKING' unit has a objNull target, they drop out of the BIS attack fsm.